recipes/list_items/contact_info/contact_info.stories.js

```diff
@@ -108,9 +108,11 @@ export const Default = {
   render: DefaultTemplate,
 
   args: {
+    showAvatar: true,
     avatarSrc: avatarImage,
     avatarSeed: 'seed',
     avatarInitials: 'JL',
+    avatarIcon: 'user',
```

Comment:
This is not needed as `avatarIcon` prop is `user` by default

Reply:
Hi @juliodialpad , after I remove this line, I found the `avatarIcon` field become empty string in the panel:
![Screenshot 2023-05-16 at 11 50 42 AM](https://github.com/dialpad/dialtone-vue/assets/61763780/923a90fb-2690-4495-a714-fa1a9a05e816)

Reply:
That's intended, that means that if no `avatarIcon` prop is set 'user' will be used as default.

Actually, please update that field to use the list of icons. You can see an example of that over [icon.stories.js](https://github.com/dialpad/dialtone-vue/blob/83c3c3ad37968ee5beaf953c70b986529b626199/components/icon/icon.stories.js#L6)

Reply:
Done, please review:
![Screenshot 2023-05-18 at 12 16 19 PM](https://github.com/dialpad/dialtone-vue/assets/61763780/66fb6cbd-0d98-4ce2-80c4-eea711a416ac)